### DISCUSSION TITLE
"Error: VT13001: [BUG] in scatter query: complex ORDER BY expression" Whilst trying to push to database with Prisma

### DISCUSSION THREAD

**Discussion by tobyab** [planetscale/discussion#487] [Q&A]
Hey! I'm trying to push to my PlanetScale Database from Prisma. However, I keep getting this error. Here's the full stack trace:
```
Error: VT13001: [BUG] in scatter query: complex ORDER BY expression: case when table_info.create_options = :table_info_create_options /* VARCHAR */ then true else false end
   0: sql_migration_connector::flavour::mysql::connection::describe_schema
             at migration-engine/connectors/sql-migration-connector/src/flavour/mysql/connection.rs:34
   1: migration_core::commands::schema_push::Calculate `from`
             at migration-engine/core/src/commands/schema_push.rs:46
   2: migration_core::state::SchemaPush
             at migration-engine/core/src/state.rs:433
```
Both Prisma, and Prisma client are version 4.15.0, and my relationMode is set to Prisma in my schema. Let me know if you need any further details, or if you want me to elaborate. I'd really appreciate some help on this! :)

**Comment by phbernard** [comment]
Same issue here. I'm trying to push my updated schema, but today I get the same error message. Although I updated my schema since the last time I pushed (... thus the push), I don't think the change is so special that it could raise a error message I managed to dodge until now.

I can push the same schema on a self-hosted MySQL database.

The freshness of this issue (here and [there](https://github.com/prisma/prisma/issues/19645)) makes me think this could be a new behavior on PlanetScale.

**Reply by tobyab** [reply]
Yeah. I imagine this is down to Vitess - that could explain why you're able to push to a self hosted instance of MySQL.

**Comment by phbernard** [comment]
Updating Prisma to 4.15.0 worked for me.

**Reply by tobyab** [reply]
Ah, I think I've found my problem! My lock file is telling me that Prisma v15.4.0 is installed - even though it's not. I'm guessing this is down to global installations.
